src/iqm/qiskit_iqm/fake_backends/fake_adonis.py

```diff
@@ -20,30 +20,35 @@
 def IQMFakeAdonis() -> IQMFakeBackend:
     """Return IQMFakeBackend instance representing IQM's Adonis architecture."""
     architecture = QuantumArchitectureSpecification(
-        name="Adonis",
-        operations=["phased_rx", "cz", "measurement", "barrier"],
-        qubits=["QB1", "QB2", "QB3", "QB4", "QB5"],
-        qubit_connectivity=[["QB1", "QB3"], ["QB2", "QB3"], ["QB3", "QB4"], ["QB3", "QB5"]],
+        name='Adonis',
+        operations={
+            'prx': [['QB1'], ['QB2'], ['QB3'], ['QB4'], ['QB5']],
+            'cz': [['QB1', 'QB3'], ['QB2', 'QB3'], ['QB4', 'QB3'], ['QB5', 'QB3']],
+            'measure': [['QB1'], ['QB2'], ['QB3'], ['QB4'], ['QB5']],
+            'barrier': [],
+        },
+        qubits=['QB1', 'QB2', 'QB3', 'QB4', 'QB5'],
+        qubit_connectivity=[['QB1', 'QB3'], ['QB2', 'QB3'], ['QB3', 'QB4'], ['QB3', 'QB5']],
     )
     error_profile = IQMErrorProfile(
-        t1s={"QB1": 27000.0, "QB2": 33000.0, "QB3": 25000.0, "QB4": 40000.0, "QB5": 25000.0},
-        t2s={"QB1": 20000.0, "QB2": 26000.0, "QB3": 23000.0, "QB4": 26000.0, "QB5": 7000.0},
+        t1s={'QB1': 27000.0, 'QB2': 33000.0, 'QB3': 25000.0, 'QB4': 40000.0, 'QB5': 25000.0},
+        t2s={'QB1': 20000.0, 'QB2': 26000.0, 'QB3': 23000.0, 'QB4': 26000.0, 'QB5': 7000.0},
         single_qubit_gate_depolarizing_error_parameters={
-            "phased_rx": {"QB1": 0.0006, "QB2": 0.0054, "QB3": 0.0001, "QB4": 0.0, "QB5": 0.0005}
+            'prx': {'QB1': 0.0006, 'QB2': 0.0054, 'QB3': 0.0001, 'QB4': 0.0, 'QB5': 0.0005}
```

Comment:
I noticed now that currently the code base has used both single and double quotes for strings. Linter requires that the same quotes are used within a file, but not across the entire project. I changed these because I had to fix this file and the other files I worked with had used single quotes.